src/App.jsx

```diff
@@ -43,16 +44,21 @@ export function App() {
 			<Route path="/" element={<Layout />}>
 				<Route
 					index
-					element={<Home 
-					handleClick={handleClick} 
-					listToken={listToken} 
-					displayName={displayName}
-					    handleInputChange={handleInputChange}
-					    setDisplayName={setDisplayName}
-					    setListToken={setListToken}
-				/>}
+					element={
+						<Home
+							handleClick={handleClick}
+							listToken={listToken}
+							displayName={displayName}
+							handleInputChange={handleInputChange}
+							setDisplayName={setDisplayName}
+							setListToken={setListToken}
```

Comment:
I know that you didn't do these changes in this PR, but what is the difference between `displayName` and `listToken`? To me it seems that both refer to the list token?

Reply:
I think `displayName` is handling something that the other team worked on in their previous PR. The last time we did not set it, it broke `main`.